packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -1016,6 +1017,14 @@ const api = new apigateway.SpecRestApi(this, 'ExampleRestApi', {
 [`x-amazon-apigateway-endpoint-configuration`](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-endpoint-configuration.html) 
 in your openApi file. 
 
+## Metrics
```

Comment:
I'm curious why you're only interested in metrics across all APIs in the account/region. Isn't it more useful to measure, graph or alarm these metrics for specific APIs?

Reply:
The metric emitted by apigateway will be for all the APIs. These can then be filtered in the cloudwatch console by ApiName, Method, Resource etc. since these are the dimensions of these metirc. 
https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-metrics-and-dimensions.html

Reply:
Why not add these as instance methods to `RestApi` and add the `restApiName` as a dimension already? This feels more useful to you and others, than a CloudWatch metric that points at metrics of all Rest APIs in the account/region, no?

I'm curious what your use case is.

Reply:
Initially I was trying to model it in the way other resources had it modelled (eg. Lambda). Now that I think about it, it does indeed makes sense to have them bound to one particular API instead of all.

Thanks for this suggestion. Let me make it.